packages/ui/package.json

```diff
@@ -14,6 +14,7 @@
     "prepack": "npm run build"
   },
   "dependencies": {
+    "@fontsource-variable/inter": "^5.0.8",
```

Comment:
Are we pulling in this font as it is an exact match for figma?

Do we have the option of inlining it - rather than pulling in a dep?

Reply:
@kanej this font is the one from figma, yes. I did some light research into importing fonts with vite and this seemed to be the easiest solution I found.